mlir/include/mlir/IR/OpAsmInterface.td

```diff
@@ -109,6 +109,31 @@ def OpAsmOpInterface : OpInterface<"OpAsmOpInterface"> {
   ];
 }
 
+//===----------------------------------------------------------------------===//
+// OpAsmTypeInterface
+//===----------------------------------------------------------------------===//
+
+def OpAsmTypeInterface : TypeInterface<"OpAsmTypeInterface"> {
+  let description = [{
+    This interface provides hooks to interact with the AsmPrinter and AsmParser
```

Comment:
Micro nit: This type interface 

I was just thinking the same description could probably have been used for 2 other interfaces, and I don't recall if our documentation generation differntiates between interface types well enough that it would be shown there.